controllers/cloudinit/controlplane_init.go

```diff
@@ -141,7 +143,13 @@ func NewInitControlPlane(input *ControlPlaneInitInput) (*CloudConfig, error) {
 		fmt.Sprintf("%s %q %q", scriptPath(snapstoreHTTPProxyScript), input.SnapstoreHTTPProxy, input.SnapstoreHTTPSProxy),
 		fmt.Sprintf("%s %q %q", scriptPath(snapstoreProxyScript), input.SnapstoreProxyDomain, input.SnapstoreProxyId),
 		scriptPath(disableHostServicesScript),
-		fmt.Sprintf("%s %q", scriptPath(installMicroK8sScript), installArgs),
+		fmt.Sprintf("%s %q", scriptPath(installMicroK8sScript), installArgs))
```

Comment:
move that close paren to a new line so it's easier to see

```suggestion
		fmt.Sprintf("%s %q", scriptPath(installMicroK8sScript), installArgs)
    )
```